storybookjs/storybook
# Use community AsyncStorage instead of deprecated React Native one

Issue: #6078

## What I did

Added a dependency on [@react-native-community/async-storage](https://github.com/react-native-community/react-native-async-storage) to replace the deprecated one from RN, which was causing warnings to appear when using RN 0.59 (see https://github.com/storybooks/storybook/issues/6078).

## How to test

Run the kitchen sink app and there's no longer a warning displayed.



It would be brilliant if this could make it into both the 5.x release and also a 4.1.x patch release, as it'll be affecting people using RN 0.59 who aren't able to migrate to Storybook 5 yet.
